.flake8

```diff
@@ -16,9 +16,11 @@ ignore =
     D401
     E126
     E203
-    I005
+    F722
     I001
+    I005
```

Comment:
" isort found an unexpected missing import". I believe this is dangerous to ignore, don't you agree?

Reply:
Let's check it in another PR.